### DISCUSSION TITLE
PlutoGridEnterKeyAction.editingAndMoveDown

### DISCUSSION THREAD

**Discussion by mohit-2003** [bosskmk/pluto_grid#566] [Q&A]
`configuration: const PlutoGridConfiguration( enterKeyAction: PlutoGridEnterKeyAction.editingAndMoveDown) `
**This is not working in Android when pressing enter key.**

**Comment by bosskmk** [comment]
@mohit-2003 
Can you attach an example code to reproduce the problem?

**Reply by mohit-2003** [reply]
`configuration: const PlutoGridConfiguration( enterKeyAction: PlutoGridEnterKeyAction.editingAndMoveDown)` This is default in PlutoGrid constructor.

https://user-images.githubusercontent.com/86219502/188255612-85017ad1-891e-4312-95d6-698f4c29f210.mp4

**Comment by bosskmk** [comment]
@mohit-2003 
I know your problem.
thank you

**Reply by mohit-2003** [reply]
Is there any solution ?

**Comment by bosskmk** [comment]
@mohit-2003 
For mobile, the code must be modified.
I will fix it in the next version.

**Reply by mohit-2003** [reply]
Thanks, I'm waiting for update 😊
